lib/urlwatch/reporters.py

```diff
@@ -175,48 +166,98 @@ def _parts(self):
             yield SafeHtml('<hr>')
 
         yield SafeHtml("""
-        <address>
-        {pkgname} {version}, {copyright}<br>
-        Website: {url}<br>
-        watched {count} URLs in {duration} seconds
-        </address>
-        </body>
-        </html>
-        """).format(pkgname=urlwatch.pkgname, version=urlwatch.__version__, copyright=urlwatch.__copyright__,
-                    url=urlwatch.__url__, count=len(self.job_states), duration=self.duration.seconds)
-
-    def _diff_to_html(self, unified_diff):
-        for line in unified_diff.splitlines():
-            if line.startswith('+'):
-                yield SafeHtml('<span class="unified_add">{line}</span>').format(line=line)
-            elif line.startswith('-'):
-                yield SafeHtml('<span class="unified_sub">{line}</span>').format(line=line)
-            else:
-                yield SafeHtml('<span class="unified_nor">{line}</span>').format(line=line)
+            <div style="font-family:monospace;font-style:italic">
+            <span>{pkgname} {version}, {copyright}</span>
+            <address>Website: <a href="{url}">{url}</a></address>
+            <span>Watched {count} URLs in {duration} seconds</span>
+            </div>
+            </body>
+            </html>""").format(pkgname=urlwatch.pkgname, version=urlwatch.__version__, copyright=urlwatch.__copyright__,
+                               url=urlwatch.__url__, count=len(self.job_states), duration=self.duration.seconds)
+
+    def _diff_to_html(self, unified_diff, is_markdown=False):
```

Comment:
Since this function isn't using the `self` object, it could be made a stateless global function (`def _diff_to_html(unified_diff, is_markdown=False);` in module-global scope) and have some basic unit tests added.

Reply:
Sorry, don't have the time to refactor existing code.  This contribution is already costing me way too much,

Reply:
> Sorry, don't have the time to refactor existing code. This contribution is already costing me way too much,

Well I guess then I am not going to spend my time to review your PR. Feel free to maintain this thing in your fork then.

Reply:
Will do, thanks.